sdk/storage/azure-storage-blob/azure/storage/blob/_shared/avro/datafile.py

```diff
@@ -107,11 +107,16 @@ def __init__(self, reader, datum_reader, **kwargs):
         # get ready to read
         self._block_count = 0
 
+        # event_position and block_count are to support reading from current position in the future read,
+        # no need to downloading from the beginning of avro file with these two attr.
+        if hasattr(self._reader, 'event_position'):
```

Comment:
I'd rename this to `object_position` or so. avro shouldn't be aware of CF and `event` is CF terminology.